HashRegistrarSimplified.sol

```diff
@@ -412,12 +434,11 @@ contract Registrar {
      * @param unhashedName An invalid name to search for in the registry.
      * 
      */
-    function invalidateName(string unhashedName) {
+    function invalidateName(string unhashedName) inState(sha3(unhashedName), Mode.Owned) {
```

Comment:
Again, if a name could be invalidated before auctions take place, then bids would be returned.